lib/Common/Common/SmartFpuControl.cpp

```diff
@@ -16,7 +16,7 @@
 static errno_t SetFPUControlDefault(void)
 {
 #if _WIN32
-#if _M_AMD64 || _M_ARM
+#if _M_AMD64 || _M_ARM || _M_ARM64
     return _controlfp_s(0, _RC_NEAR + _DN_SAVE + _EM_INVALID + _EM_ZERODIVIDE +
```

Comment:
Why did you add || _M_ARM64 here instead of converting to !defined(_M_IX86) like above?